app/src/main/java/com/duckduckgo/app/fire/fireproofwebsite/ui/FireproofWebsitesViewModel.kt

```diff
@@ -59,6 +60,7 @@ class FireproofWebsitesViewModel @Inject constructor(
 
     private val fireproofWebsites: LiveData<List<FireproofWebsiteEntity>> = fireproofWebsiteRepository.getFireproofWebsites()
     private val fireproofWebsitesObserver = Observer<List<FireproofWebsiteEntity>> { onPreservedCookiesEntitiesChanged(it!!) }
+    private var removedWebsitesEntities: List<FireproofWebsiteEntity> = emptyList()
```

Comment:
Could we remove keeping this state by passing the list along with the command, then just pass these back to `ViewModel` in the undo action in the `Snackbar`?

Something like: https://github.com/duckduckgo/Android/blob/b5dbb62bd910ecb76a207fe31371b9d1224783a7/app/src/main/java/com/duckduckgo/app/bookmarks/ui/BookmarksActivity.kt#L326